src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValue.cs

```diff
@@ -43,5 +44,16 @@ internal FormulaValue(IRContext irContext)
         public abstract object ToObject();
 
         public abstract void Visit(IValueVisitor visitor);
+
+        public abstract void ToExpression(StringBuilder sb);
+
+        public string ToExpression()
```

Comment:
>public string ToExpression() [](http://example.com/codeflow?start=8&length=28)

Can we add an xml comment?

/// Provides serialization. Return an expression that, when evaluated, recreates an equivalent formula value, including its type. 
/// This may not be the same as the expression used to originally create this type.